### DISCUSSION TITLE
is it possible to render Pluto notebook non-interactively?

### DISCUSSION THREAD

**Discussion by rczerminski-valo** [fonsp/Pluto.jl#1594] [Questions about using Pluto] [answered]
Is it possible to render Pluto notebook non-interactively? i.e. instead starting the server Pluto.run() and pressing render symbol (triangle/circle) I wonder if there is a way to do something like Pluto.render(FILE) where FILE is e.g. "test.html". This would be very useful for testing etc.

**Comment by Pangoraw** [comment, marked as answer]
You can use [PlutoSliderServer.jl](https://GitHub.com/JuliaPluto/PlutoSliderServer.jl) and its [`export_notebook(notebook_file)`](https://github.com/JuliaPluto/PlutoSliderServer.jl/blob/60f5ee0fb6fcfa23a9a7f4cd03c9bcc45338413f/src/PlutoSliderServer.jl#L75) function to automatically export a notebook to html.
